vir/defs/polymorphic/ast/stmt.rs

```diff
@@ -167,6 +169,18 @@ impl fmt::Display for Assert {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct Assume {
```

Comment:
Why not re-use `inhale` for the assumptions?

Reply:
Done!